experimental/GModule/Brueckner.jl

```diff
@@ -458,6 +474,11 @@ function lift(C::GModule, mp::Map)
   return allG
 end
 
+function solvable_quotient(G::Oscar.GAPGroup)
+  q = pc_group(symmetric_group(1))
+  mp = hom(G, q, [one(q) for g = gens(G)])
```

Comment:
My personal preference
```suggestion
  mp = hom(G, q, [one(q) for g in gens(G)])
```